wagtail/contrib/redirects/views.py

```diff
@@ -36,65 +36,91 @@
 )
 from wagtail.log_actions import log
 
-permission_checker = PermissionPolicyChecker(permission_policy)
 
+class RedirectPageColumn(Column):
+    cell_template_name = "wagtailredirects/redirect_page.html"
 
-@permission_checker.require_any("add", "change", "delete")
-@vary_on_headers("X-Requested-With")
-def index(request):
-    query_string = request.GET.get("q", "")
-    ordering = request.GET.get("ordering", "old_path")
+    def get_cell_context_data(self, instance, parent_context):
+        context = super().get_cell_context_data(instance, parent_context)
+        context["redirects"] = instance
+        return context
```

Comment:
The `instance` is already available as the `instance` variable in the context from the `Column.get_cell_context_data`

```suggestion
```

Reply:
It looks like this wasn't committed yet?...

Reply:
Sorry, I forgot to make this change - it's fixed now.